www/userguide/userguide.md

```diff
@@ -0,0 +1,254 @@
+\page userguide User Guide
+\tableofcontents
+
+**CodeChecker** is a static analysis infrastructure built on the [LLVM/Clang
+Static Analyzer](http://clang-analyzer.llvm.org) toolchain, replacing
+[`scan-build`](http://clang-analyzer.llvm.org/scan-build.html) in a Linux or
+macOS (OS X) development environment.
+
+\section userguide_products Products
+The product system allows a single CodeChecker server to serve multiple separate
+result databases, named "products", under the same IP address and authentication
+domain.
+
+\subsection userguide_managing_products Managing products
+![Products](images/userguide/products.png)
+
+After enabling the administrative actions in the top right corner, click
```

Comment:
What do "administrative actions" mean? This term is used several times in the text without its meaning being defined.